src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -39,6 +39,31 @@ To forward your Kubernetes logs to New Relic with our plugin:
 
 <InstallFeedback />
 
+## Install Fluent Bit output plugin [#fluent-bit]
```

Comment:
@paperclypse how is this different from our separate plug-in doc about Fluent Bit: https://docs.newrelic.com/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding/

Why is Fluent Bit being singled out here in this context? Why aren't we mentioning other log forwarders here? 

Previously, this Kubernetes plug-in doc was fine without a reference to the Fluent Bit plug-in. What changed?